Test/comp/Class.dfy

```diff
@@ -19,7 +20,7 @@ class MyClass {
   function method F(): int { 8 }
   static function method G(): int { 9 }
   method M() returns (r: int) { r := 69; }
-  static method N() returns (r: int) { return 70; }
+  static method N() returns (r: int) { r := 70; }
```

Comment:
Fair, but I'd rather optimize for coverage in test cases rather than nice compiler output (especially tailored to a particular target language).

The better change IMO would be just to change one of them, for better coverage. If anything if one option produces weird/dumb code, all the more reason to make sure it still executes correctly. :)

Reply:
This isn't a special python thing. I'm also fairly confident that we test this elsewhere, this file is about Classes after all.

Reply:
It's also more idiomatic Dafny.

Reply:
I wasn't actually that confident we test that in any other test files currently run on Python, since the coverage is low so far. I did a quick search now and found a couple of return statements though.

I'm fine with this change in this case. But I wanted to be clear that making the test cases idiomatic is largely an anti-requirement - better to make sure we have good coverage of both idiomatic and ugly Dafny code. :)